wp-youtube-lyte.php

```diff
@@ -635,6 +638,9 @@ function lyte_init() {
 
     echo "<script type=\"text/javascript\">var bU='" . $lyteSettings['path'] . "';" . $mobJS . $doublecheck_thumb_cookie . "style = document.createElement('style');style.type = 'text/css';rules = document.createTextNode(\"".$lyte_css."\" );if(style.styleSheet) { style.styleSheet.cssText = rules.nodeValue;} else {style.appendChild(rules);}document.getElementsByTagName('head')[0].appendChild(style);</script>";
     echo "<script type=\"text/javascript\" async src=\"".$lyteSettings['path'].$lyteSettings['file']."\"></script>";
+    if ( apply_filters( 'lyte_use_internal_lazyloader', true ) ) {
```

Comment:
use from lyteSettings instead of re-checking the filter (see earlier comment)?

Reply:
Done in latest commit.

Reply:
Afraid that at the point where you added the filter now, it triggers much to early (as the setting of lyteSettings is done immediately, not hooking into any wordpress core action). Best would be to add it in the `lyte_settings_enforcer` function right before `lyte_settings` filter is called which hooks into `after_setup_theme`?